src/pylon_camera/main.cpp

```diff
@@ -35,19 +35,34 @@
 #include <ros/ros.h>
 #include <boost/thread.hpp>
 #include <pylon_camera/pylon_camera_node.h>
+#include <pylon_camera/CameraSettingsConfig.h>
+
+void callback(pylon_camera::CameraSettingsConfig &config, uint32_t level) {
+	ROS_INFO_STREAM("dynamic_reconfigure callback");
+}
 
 
 int main(int argc, char **argv)
 {
     ros::init(argc, argv, "pylon_camera_node");
+
+
     ros::NodeHandle nh_private("~");
     ros::NodeHandle nh_image;
 
+	dynamic_reconfigure::Server<pylon_camera::CameraSettingsConfig> server;
+    dynamic_reconfigure::Server<pylon_camera::CameraSettingsConfig>::CallbackType f;
```

Comment:
This shouldn't be in `main()` now, right? You moved it to the PylonCameraNode class.